src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -108,11 +108,12 @@ export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = (
 
   return (
     <>
-      {/* TODO: Update meta tags */}
       <MetaTags
-        pathname="my-collection"
-        title="Artwork | My Collection | Artsy"
-        description="..."
```

Comment:
not sure if we need the `pathname` and the `description` here

Reply:
I don't think we need them here because this will most likely be visible.